dev/TabView/TabView.xaml

```diff
@@ -420,8 +431,9 @@
                                 MaxWidth="{ThemeResource TabViewItemHeaderIconSize}"
                                 MaxHeight="{ThemeResource TabViewItemHeaderIconSize}"
                                 Margin="{ThemeResource TabViewItemHeaderIconMargin}">
-                                <ContentPresenter x:Name="Icon"
-                                    Content="{TemplateBinding Icon}"
+                                <ContentControl x:Name="IconControl"
```

Comment:
Something like this would be useful just in general, in terms of a control that you can give an IconSource and it'll put the resulting IconElement into the UI tree.  I imagine that, given that this has already gone through API review, now wouldn't be the time to address that, but we should file an issue if we don't already have one saying that we should make a control like that.  I don't imagine it'd be a big work item, and it'd be a handy utility.  We had one of those in the OS repo, but it was pulled prior to 19H1 due to not having gone through API review in time.

Reply:
Jevan actually started a thread about this, so I'll await the results -- but agreed, it's not very attractive as is.